pkg/datastore/documentloader/pdf.go

```diff
@@ -1,15 +1,15 @@
 package documentloader
 
 import (
+	"bytes"
 	"context"
-	"fmt"
-	"io"
-	"os"
 	"strings"
 
 	"github.com/gptscript-ai/knowledge/pkg/datastore/types"
 	vs "github.com/gptscript-ai/knowledge/pkg/vectorstore"
 	"github.com/ledongthuc/pdf"
```

Comment:
This can be removed then, right? 
Though personally I'd rather have both unidoc and this available to choose 🤷‍♂️

Reply:
Hmm yeah, My take is for simplity just use the better one for now. The old one did fail quite a lot of tests, and also not able to load some of the pdf documents from benchmark though.

Reply:
It's really bugging me, that there is no good MIT/Apache/actually-freen-and-open licensed PDF parser in Go... Tempting to fix the one we were using before

Reply:
Yeah I agree. Even the one i am using had some problems parsing the text. We will definetely revisit this part